### ISSUE TITLE
Just wanted to say thanks <3

### ISSUE THREAD

**Issue by tchakabam** [wrigby/flowdump#6]
your tool saved my day

**Comment by wrigby** [comment]
I'm so glad! Also, thanks a ton for dropping by. 

You may also want to look at tcpflow (https://github.com/simsong/tcpflow) - I realized recently that I wouldn't have needed to write my own if I had known about tcpflow at the time.

**Comment by tchakabam** [comment]
Yes, tcpflow includes what this here does eventually, but honestly I found your tool *much* more straightforward to use :) 

EDIT: Ah no I meant `tcpreplay` - but anyhow, sometimes smaller more focused tools are better. 

Thanks for the pointer on tcpflow. 

>  I realized recently that I wouldn't have needed to write my own if I had known about tcpflow at the time.

Classic :)
